data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/AddEntryProcessorConfig.java

```diff
@@ -21,6 +21,10 @@ public static class Entry {
         @NotNull
         private Object value;
 
+        @NotEmpty
```

Comment:
I see `@NotEmpty` Jakarta validation on both. This will throw an error if you don't configure them. Maybe you should custom validation here without using `@NotEmpty`.

Reply:
Updated the validation and the documentation.